dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -237,14 +240,101 @@ std::wstring Microsoft::Windows::AppNotifications::Helpers::GetDisplayNameBasedO
     return displayName;
 }
 
-// Placeholder
-HRESULT RetrieveAssetsFromProcess(_Out_ Microsoft::Windows::AppNotifications::Helpers::AppNotificationAssets& /*assets*/)
+#define GIL_EXACTSIZEONLY       0x0100
+// Looks at the passed in icon and tries loading a better sized icon from the resource file.
```

Comment:
How much of this comment is relevant to us?

Reply:
Will be discussed tomorrow and follow it up with a new update.